lib/abide_dev_utils/cli/xccdf.rb

```diff
@@ -43,10 +43,11 @@ def execute(xccdf_file)
         end
         @data[:console] = true if @data[:dir].nil?
         @data[:type] = 'cis' if @data[:type].nil?
+        @data[:parent_key_prefix] = @data[:prefix]
         @data[:parent_key_prefix] = '' if @data[:parent_key_prefix].nil?
         hfile = AbideDevUtils::XCCDF.gen_map(xccdf_file, **@data)
         mapping_dir = File.dirname(hfile.keys[0]) unless @data[:dir].nil?
-        AbideDevUtils::Output.simple("Creating directory #{mapping_dir}") unless @data[:quiet] || @data[:console] || @data[:dir].nil?
+        AbideDevUtils::Output.simple("Creating directory #{mapping_dir}") unless @data[:quiet] || @data[:console] || @data[:dir].nil? || File.directory?(mapping_dir)
```

Comment:
Adding the `File.directory?` conditional here doesn't actually stop the dir creation attempt, it just stops the output.

Reply:
Which is what I wanted.